bcda/main_test.go

```diff
@@ -213,47 +310,69 @@ func (s *MainTestSuite) TestCreateAlphaTokenCLI() {
 	buf := new(bytes.Buffer)
 	s.testApp.Writer = buf
 
+	assert := assert.New(s.T())
+
+	outputPattern := regexp.MustCompile(`[a-zA-Z]+, \d+-[a-zA-Z]{3}-\d{2} \d{2}:\d{2}:\d{2} [A-Z]+\n[a-fA-F0-9]{8}(?:-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}\n.+\..+\..+\n`)
+
 	// execute positive scenarios via CLI
 	args := []string{"bcda", "create-alpha-token", "--ttl", "720", "--size", "Dev"}
 	err := s.testApp.Run(args)
-	assert.Nil(s.T(), err)
-	assert.NotNil(s.T(), buf)
+	assert.Nil(err)
+	assert.Regexp(outputPattern, buf.String())
 	buf.Reset()
 
 	// ttl is optional when using the CLI
 	args = []string{"bcda", "create-alpha-token", "--size", "Dev"}
 	err = s.testApp.Run(args)
-	assert.Nil(s.T(), err)
-	assert.NotNil(s.T(), buf)
+	assert.Nil(err)
+	assert.Regexp(outputPattern, buf.String())
+	buf.Reset()
+
+	args = []string{"bcda", "create-alpha-token", "--size", "DEV"}
+	err = s.testApp.Run(args)
+	assert.Nil(err)
+	assert.Regexp(outputPattern, buf.String())
 	buf.Reset()
 
 	// Execute CLI with invalid inputs
+	args = []string{"bcda", "create-alpha-token"}
+	err = s.testApp.Run(args)
+	assert.Equal("invalid argument for --size.  Please use 'Dev', 'Small', 'Medium', or 'Large'", err.Error())
```

Comment:
Should we test all cases - dev, small, medium, and large?

Reply:
We have test calls to `createAlphaToken()` with all the sizes in these functions: https://github.com/CMSgov/bcda-app/blob/0addc523b503f1e97fe433779185d64d62f9889c/bcda/main_test.go#L685

Reply:
Excellent!